### ISSUE TITLE
service.list fails when no services are deployed

### ISSUE THREAD

**Issue by mdantonio** [gabrieldemarmiesse/python-on-whales#302]
After further tests on #300 I found a regression:

in swarm mode when no services are deployed, `service.list()` fails

This is because 

```
        full_cmd = self.docker_cmd + ["service", "list", "--quiet"]

        ids_truncated = run(full_cmd).splitlines()
```

produces an empty list and then the following command is wrongly composed

```
        full_cmd = (
            self.docker_cmd
            + ["service", "inspect"]
            + ids_truncated
            + ["--format", "{{.ID}}"]
        )
```

=>

```
The docker command executed was `/usr/bin/docker service inspect --format {{.ID}}`.
It returned with code 1
The content of stdout is ''
The content of stderr is '"docker service inspect" requires at least 1 argument.
See 'docker service inspect --help'.

Usage:  docker service inspect [OPTIONS] SERVICE [SERVICE...]

Display detailed information on one or more services
```

I think that the problem can be fixed by simply adding:

```
        if not ids_truncated:
            return []
```
(like here: https://github.com/mdantonio/python-on-whales/commit/7c9fac08b88399ae2219159c094a6ef08fd653cc)

**Comment by gabrieldemarmiesse** [comment]
Oops. I see what caused it. Thanks for pointing that out. Time for more
unit tests and a quick release!

**Comment by mdantonio** [comment]
I confirm the fix with 0.36.1
